src/workerd/server/workerd-api.c++

```diff
@@ -354,10 +372,47 @@ void WorkerdApi::compileModules(
   lockParam.withinHandleScope([&] {
     auto modules = jsg::ModuleRegistryImpl<JsgWorkerdIsolate_TypeWrapper>::from(lockParam);
 
+    if (util::Autogate::isEnabled(util::AutogateKey::BUILTIN_WASM_MODULES) &&
+        conf.getModules().size() > 0) {
```

Comment:
Shouldn't this be doing some sort of check to see if the worker uses Python before injecting Pyodide stuff?

Reply:
oops, yeah.